py/tests/test_emails.py

```diff
@@ -476,6 +476,119 @@ async def test_send_event_update(cli, url, login, factory: Factory, dummy_server
+    await factory.create_cat()
+    await factory.create_user()
+    evt = await factory.create_event(price=10)
+    tt = await db_conn.fetch('SELECT id FROM ticket_types WHERE event=$1', evt)
+
+    await login()
+
+    anne = await factory.create_user(first_name='anne', email='anne@example.org')
+    await factory.buy_tickets(await factory.create_reservation(anne, None, ticket_type_id=tt[0]['id']))
+    assert len(dummy_server.app['emails']) == 1
+
+    mary = await factory.create_user(first_name='mary', email='mary@example.org')
+    await factory.buy_tickets(await factory.create_reservation(mary, None, ticket_type_id=tt[1]['id']))
+    assert len(dummy_server.app['emails']) == 2
+
+    data = dict(
+        {
+            "subject": 'This is a test',
+            "message": 'this is the default **message**.',
+            "groupMessages": [{'ticketType': tt[1]['id'], 'message': 'this is the first **message**.'}],
+        }
+    )
+
+    r = await cli.json_post(url('event-send-update', id=factory.event_id), data=data)
+    assert r.status == 200, await r.text()
+
+    emails = dummy_server.app['emails']
+    assert len(dummy_server.app['emails']) == 4
+
+    email = next((e for e in emails if 'the first' in e['part:text/html']), False)
+    assert email
+    assert email['Subject'] == 'This is a test'
+    assert email['To'] == 'mary Spencer <mary@example.org>'
+    assert '<p>Hi mary,</p>\n\n<p>this is the first <strong>message</strong>.</p>\n' in email['part:text/html']
+
+    email = next((e for e in emails if 'the default' in e['part:text/html']), False)
+    assert email
+    assert email['Subject'] == 'This is a test'
+    assert email['To'] == 'anne Spencer <anne@example.org>'
+    assert '<p>Hi anne,</p>\n\n<p>this is the default <strong>message</strong>.</p>\n' in email['part:text/html']
+
+
+async def test_send_event_update_multiple_messages(cli, url, login, factory: Factory, dummy_server, db_conn):
+    await factory.create_company()
+    await factory.create_cat()
+    await factory.create_user()
+    evt = await factory.create_event(price=10)
+    tt = await db_conn.fetch('SELECT id FROM ticket_types WHERE event=$1', evt)
+
+    await login()
+
+    anne = await factory.create_user(first_name='anne', email='anne@example.org')
+    await factory.buy_tickets(await factory.create_reservation(anne, None, ticket_type_id=tt[0]['id']))
+    assert len(dummy_server.app['emails']) == 1
+
+    mary = await factory.create_user(first_name='mary', email='mary@example.org')
+    await factory.buy_tickets(await factory.create_reservation(mary, None, ticket_type_id=tt[1]['id']))
+    assert len(dummy_server.app['emails']) == 2
+
+    jane = await factory.create_user(first_name='jane', email='jane@example.org')
+    await factory.buy_tickets(await factory.create_reservation(jane, None, ticket_type_id=tt[2]['id']))
+    assert len(dummy_server.app['emails']) == 3
+
+    data = dict(
+        {
+            "subject": 'This is a test',
```

Comment:
```suggestion
            'subject': 'This is a test',
```
Please make consistent use of single quotes and double quotes: single quotes unless the string has single quotes in it.